src/pages/questionPage.js

```diff
@@ -75,6 +76,9 @@ const nextQuestion = () => {
   if (quizData.currentQuestionAnswer === null) {
     const alertElement = createAlertElement();
```

Comment:
It can still create multiple alert elements. Maybe using getElementById you can check if there is already an alert elements, if there is already one, don't add a second one